crates/tuono_lib/src/lib.rs

```diff
@@ -16,6 +17,7 @@ mod ssr;
 mod vite_reverse_proxy;
 mod vite_websocket_proxy;
 
+pub use env::EnvVarManager;
```

Comment:
We should not expose the `EnvManager` to the userland
```suggestion
use env::EnvVarManager;
```